### PR TITLE
ignore empty lines in redirect

### PR DESCRIPTION
**Pull request by shreyanshp**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ async function parseRedirects(event) {
 function checkRedirect(request) {
   const url = new URL(request.url).pathname
   for (const [pattern, redirectUrl] of redirectMap) {
-    if (url.match(pattern)) {
+    if (url.match(pattern) && pattern != "") {
```
Comment (eramus) [inline index.js:36]: Itd be better to check if the string is empty or not before performing the regex.

**Review by PorlyBe — APPROVED** [review]

**Review by eramus — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (shreyanshp) [reply index.js:36]: I think this should suffice, if not please feel free to raise a PR!

Comment (eramus) [reply index.js]: Raise a PR to fix this PR?

Comment (shreyanshp) [reply index.js]: done!

**Review by eramus — APPROVED** [review]

**Pull request merged**